ReadMe.md

```diff
@@ -0,0 +1,17 @@
+### Readme 
+
+This application will take an old Microsoft Flight Simulator X (FSX) 
+and convert the files and textures to the newer Microsoft Flight Simulator 2020 (MSFS) format
```

Comment:
I don't know if we want to say that it actually converts textures, since it really doesn't convert anything. It adds the .dds.json files since I had some trouble with textures, but they didn't end up making any difference. I just didn't bother removing that piece of code after I tested that.

Reply:
Fixed the description here to better indicate the function of the app.  Hopefully a bit clearer. Does not convert files, just updates the structure in the JSON so MSFS can read it.